sdk/src/main/java/com/hedera/hashgraph/sdk/FeeSchedule.java

```diff
@@ -0,0 +1,60 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.threeten.bp.Instant;
+
+public class FeeSchedule {
+    private List<TransactionFeeSchedule> transactionFeeSchedules;
+    private Instant expiryTime;
+
+    public FeeSchedule() {
+        transactionFeeSchedules = new ArrayList<TransactionFeeSchedule>();
```

Comment:
Move this initializer into the field declaration; also no need to add `TransactionFeeSchedule` to the `ArrayList`'s generic parameter as it's implicitly defined by the declaration of the field.
```java
private List<TransactionFeeSchedule> transactionFeeSchedule = new ArrayList<>();
```